moninag/static/src/ts/app/contacts/contact.ts

```diff
@@ -0,0 +1,14 @@
+export class Contact {
+
+    id: number;
+    email: string;
+    first_name: string;
```

Comment:
camelCase instead of underscore according to TS code guidelines (Microsoft official convention). but that's not in the higher priority.